routes/api/ads.js

```diff
@@ -0,0 +1,89 @@
+const { Router } = require('express');
+const { Op } = require('sequelize');
+const { ad } = require('../../databases/db');
+
+const adsRouter = Router();
+const allowedFields = ['announcer', 'adType', 'redirecTo', 'img'];
```

Comment:
Desde embajadores de BDD2, agregamos el nuevo campo "description", falta tambien el campo "alias" te recomiendo revisar el modelo de ADS en el notion de BDD2

FYI @Paulaforero @HopeAero @Edusus @Nannajpg

```suggestion
const allowedFields = ['announcer', 'adType', 'redirecTo', 'img', 'description', 'alias'];
```

Reply:
A todas estas, para que es la propiedad "announcer" 🤔 ?

Reply:
Announcer era === alias xd

Reply:
Era, ya lo cambié